CASC/PortfolioMode.cpp

```diff
@@ -372,19 +344,27 @@ void PortfolioMode::getSchedules(const Property& prop, Schedule& quick, Schedule
     Schedules::getSnakeTptpSatSchedule(prop,quick);
     break;
 
-  case Options::Schedule::CASC_2024:
+  case Options::Schedule::CASC_2025:
```

Comment:
Is now the time to drop `CASC_1998` or whatever the oldest schedule is?

Reply:
It was useful to have last year, to compare against during testing. I would keep it there a bit longer. (Nothing older for CASC proper there than 2024!)

There is a different question about `getLtb2017*` but I think Josef still used it relatively recently.